schemas/com.snowplowanalytics.snowplow/custom_event/jsonschema/1-0-0

```diff
@@ -0,0 +1,35 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a custom event with generic properties",
+    "self": {
+        "vendor": "com.snowplowanalytics.snowplow",
+        "name": "custom_event",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "category": {
+            "type": ["string", "null"],
+            "maxLength": 1024,
```

Comment:
1024 feels a bit arbitrary and small, but I'm not sure if it's some sort of standard size. I would probably go with 4096, but that might be because of my lack of context

Reply:
Seems we have both 1024 and 4096 limits, so it's up to us to decide. I don't have any hard opinions